packages/xo-web/src/xo-app/jobs/new/index.js

```diff
@@ -211,8 +212,12 @@ export default class Jobs extends Component {
                 }
               } else if (group === 'host' && key === 'id') {
                 modifyProperty(property, 'Host')
-              } else if (group === 'vm' && key === 'id') {
-                modifyProperty(property, 'Vm')
+              } else if (group === 'vm') {
+                if (key === 'id') {
```

Comment:
`includes([ 'id', 'vm' ], key)`
This will fix **many** jobs forms.

Reply:
Also, please add a `modifyProperty` for host/targetHost.

Reply:
And SRs. Were those all broken or am I missing something? Maybe this requires a PR on itself to check them all.

Reply:
The `modifyProperty` for `host/targetHost/SRs` already exists
https://github.com/vatesfr/xen-orchestra/blob/88f3a0f634ebac1ad8d72df9251e3f5da1e1fbd0/packages/xo-web/src/xo-app/jobs/new/index.js#L224-L228

Reply:
Not in the "vm" group, which is an issue for [`vm.migrate`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/api/vm.js#L478) for example.